composer.json

```diff
@@ -41,7 +41,7 @@
 
     "autoload-dev": {
         "psr-4": {
-            "Tests\\Behat\\": "tests/Behat/"
+            "Tests\\": "tests/"
```

Comment:
This helps:
1. IDEs (e.g. PhpStorm or IDEA) correctly detect the tests directory as the "tests root"
2. It decrease the wtf factor if someone adds test classes elsewhere which wouldn't load otherwise
3. It may not be the best, performance-wise, but I think it's not so critical seeing as it only impacts tests (i.e. `composer --dev`).

Reply:
I wouldn't be against it, but then it does become a bit inconsistent with src:
![image](https://github.com/user-attachments/assets/0c84531f-76a3-4109-906c-6206722a432e)
(although I suppose we can do it for src too - is a change in filesystem structure considered a BC break?)

Reply:
it is not a BC break (assuming you update the autoload config to be valid for the new path), unless your project requires users to hardcode a path to a file inside your package (which is not the case for Gherkin)